packages/eas-cli/src/commands/workflow/validate.ts

```diff
@@ -43,56 +101,3 @@ export class WorkflowValidate extends EasCommand {
     }
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +100]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->maybe just throw error?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/expo/eas-cli/2700?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->